week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -26,9 +35,13 @@ The function must:
 */
 
 function remove(arr, index) {
-  return; // complete this statement
-}
+  
+  let removed=arr.spLice(3,1);
+
+      return removed; // complete this statement
 
+    }
+      
```

Comment:
Please read again carefully criteria above, particularly focusing on this:
`remove the item at the specified index`
You should use `index` function parameter that defines which element to remove.